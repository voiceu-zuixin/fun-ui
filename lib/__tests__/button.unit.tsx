import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../tsx函数式的使用例子'

describe('button tests', () => {
    it('是个div', () => {
        
        const json = renderer.create(<Button message="messdiv"/>)
            .toJSON()

        expect(json).toMatchSnapshot()
    })
})