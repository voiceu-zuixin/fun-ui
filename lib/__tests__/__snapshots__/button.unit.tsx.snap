// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button tests 是个div 1`] = `
<div>
  <h1>
    messdiv
  </h1>
  <h2>
    1
  </h2>
</div>
`;
